libbeat/docs/template-config.asciidoc

```diff
@@ -35,3 +35,17 @@ setup.template.settings:
   index.number_of_shards: 1
```

Comment:
I kind of like it is now because it matches the Elasticsearch docs where they usually talk about `index.*` as the name of the options: https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping.html